app/views/components/_navigation-sidebar.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  related_items ||= []
+  topics ||= []
+  publishers ||= []
+  collections ||= []
+  policies ||= []
+  external_links ||= []
+  related_content = [
+    {"related_items" => related_items},
+    {"topics" => topics},
+    {"publishers" => publishers},
+    {"collections" => collections},
+    {"policies" => policies},
+    {"external_links" => external_links}
+  ]
+%>
+<% if related_content.map(&:values).flatten.any? %>
+  <aside class="app-c-navigation-sidebar" role="complementary">
+    <% related_content.each do |section| %>
+      <% section.each do |section_title, links| %>
+
+        <% if section_title === "related_items" %>
+          <h2 id="nav-sidebar-<%= section_title %>" class="app-c-navigation-sidebar__main-heading" ><%= t('components.navigation_sidebar.related_content') %></h2>
+          <nav role="navigation" class="app-c-navigation-sidebar__nav-section" aria-labelledby="nav-sidebar-<%= section_title %>" %>
+            <ul>
```

Comment:
Debatable gotcha: you've not styled the UL element. Had a problem with a component recently where it found itself on a page with no CSS reset, so the UL was automatically given a margin left by Chrome. Since your component is in government-frontend this shouldn't be a problem, but maybe worth considering for future proofing.

Reply:
Good point - will add that in 👍